cnv_release_notes/cnv_release_notes.adoc

```diff
@@ -11,65 +11,91 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
 [[RN_new_features]]
 == New and changed features
 
-==== Virtual machine control from the web console
-** The updated {ProductName} web console now allows users to create virtual 
-machines, access virtual machines with either a serial or VNC connection, and 
-perform basic power management tasks (start|stop) on virtual machines. 
+=== Introducing Operators
+* Operators power the installation of KubeVirt, Containerized Data Importer, and
+the *web-ui* in {ProductName} 1.4.
```

Comment:
Is *web-ui* meant to be marked up? 
Ignore me if it is.

Reply:
I don't think it should be. Removed the asterisks.